<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>395</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gb_TimePanel">
      <property name="maximumSize">
       <size>
        <width>363</width>
        <height>186</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QTextBrowser" name="textB_results">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>200</width>
         <height>171</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>171</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="lb_Time">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>120</width>
         <height>44</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>44</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Время</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_ButtonPanel">
      <property name="maximumSize">
       <size>
        <width>363</width>
        <height>50</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb_ClearText">
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_SetCircle">
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string>Круг</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_StartStop">
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>textB_results</tabstop>
  <tabstop>pb_ClearText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
